terraform/shared/shared.tf.json.template.py

```diff
@@ -326,14 +326,37 @@ def conformance_pack(name: str) -> str:
                 }
             },
             'logs': {
+                'bucket': '${aws_s3_bucket.logs.id}',
+                'rule': [
+                    *(
+                        {
+                            'id': rule_id,
+                            'status': 'Enabled',
+                            'filter': {
+                                'prefix': filter_prefix
+                            },
+                            'expiration': {
+                                'days': config.audit_log_retention_days
+                            }
+                        }
+                        for rule_id, filter_prefix in
+                        [
+                            ('expire-alb-access', config.alb_access_log_path_prefix(deployment=None) + '/'),
+                            ('expire-s3-access', config.s3_access_log_path_prefix(deployment=None) + '/'),
+                            ('expire-bucket-inventory', config.s3_bucket_inventory_path_prefix + '/')
```

Comment:
Since the `logs` bucket is for logs, we can assume that all objects in it are logs and we don't have to restrict the expiration to specific prefixes.